examples/survival_analysis/bayes_param_survival.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    az.plot_energy(weibull_trace);
</pre><p>nitpick, can we pass <em>fill_color=('C0', 'C1') or something different to 'C5' for the second color?</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>